### PR TITLE
fix: Adding dispatcher to LoginAsync calls

### PR DESCRIPTION
**Pull request by nickrandolph**
GitHub Issue (If applicable): closes #https://github.com/unoplatform/uno.templates/issues/209

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?

Dispatcher not passed into the LoginAsync method - throws exception and fails login

## What is the new behavior?

Dipatcher passed into LoginAsync method

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.Templates/content/unoapp/MyExtensionsApp._1/Presentation/LoginModel.cs [hunk 1]
```diff
@@ -18,9 +18,9 @@ public async ValueTask Login(CancellationToken token)
         var username = await Username ?? string.Empty;
         var password = await Password ?? string.Empty;
 
-        var success = await Authentication.LoginAsync(new Dictionary<string, string> { { nameof(Username), username }, { nameof(Password), password } });
+        var success = await Authentication.LoginAsync(Dispatcher, new Dictionary<string, string> { { nameof(Username), username }, { nameof(Password), password } });
```
[the remark this answers is not in the document]
Comment (nickrandolph) [reply src/Uno.Templates/content/unoapp/MyExtensionsApp._1/Presentation/LoginModel.cs:21]: Here's not the place to change this architecture :-)

[the remark this answers is not in the document]
Comment (nickrandolph) [reply src/Uno.Templates/content/unoapp/MyExtensionsApp._1/Presentation/LoginModel.cs:21]: https://github.com/unoplatform/uno.extensions/issues/1878

**Review by dansiegel — APPROVED** [review]
Approving as is. We will need to fix the implementation to be able to inject the dispatcher in a future release of Extensions.

**Comment by nickrandolph** [comment]
@mergify backport legacy/4x release/stable/4.10

**Pull request merged**